### ISSUE TITLE
Question on Login feature

### ISSUE THREAD

**Issue by ChinmayMhatre** [tusharnankani/ToDoList#18]
would like to give the login feature a thought . would like have a conversation about it.

**Comment by tusharnankani** [comment]
Hi @ChinmayMhatre! Thanks for showing interest.

I haven't figured the layout yet. The TODOs are currently locally stored, so the user doesn't have to necessarily log in, but:
- we could maybe give a popup or something if the user wants to sign in for accessing them from multiple devices. 
- we could use `Sign In with Google`, maybe use Firebase.

Let me know if you have any questions or you have anything to add?

**Comment by ChinmayMhatre** [comment]
yeah I think that would be great . any thoughts on what kind of backend would be suitable ?

**Comment by tusharnankani** [comment]
> yeah I think that would be great . any thoughts on what kind of backend would be suitable ?

Anything would work.

**Comment by ChinmayMhatre** [comment]
Cool !  Have some college work for 1-2 weeks , will give it a look till the end of the month .

**Comment by aaryanboy** [comment]
php for the backend maybe......i would like to work on it too if this is still ongoing, anyone from 2024
